src/scenes/components/Header/Header.js

```diff
@@ -1,15 +1,55 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import styled from 'styled-components'
 
-const Container = styled.section`
-  border: 1px solid;
-  font-family: 'Roboto Mono';
+import { H3, Body2 } from '../../../components/typography'
+
+const MainSection = styled.section`
+  background-color: ${({ theme }) => theme.color('primary')}
+  color: ${({ theme }) => theme.color('neutral', 'bright')}
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 200px 10px;
+  flex-direction: column;
+`
+
+const LogoSection = styled.div`
+  margin-bottom: 50px;
+`
+
+const EventTitleSection = styled.div`
+  margin-bottom: 50px;
 `
 
-const Header = () => (
-  <Container>
-    <p>Header!</p>
-  </Container>
+const Title = styled.div`
+  text-align: center;
+  margin-bottom: 30px;
+`
+
+const DateContainer = styled.div`
+  text-align: center;
+`
+
+const Header = props => (
+  <MainSection>
+    <LogoSection>
+      <img src="https://app.zeplin.io/img/icZeplin.svg" alt="logo" />
+    </LogoSection>
+    <EventTitleSection>
+      <Title>
+        <H3>{`${props.title}`}</H3>
```

Comment:
No need for 
```js
{`${props.title}`}
```

It can be simply `{props.title}`

Reply:
(or ever `{title}` if using destructuring at the function params) -> prefered version

Reply:
👍